Update auto-generated documentation

snoopycrimecop opened the pull request:
Repository: openmicroscopy/ome-documentation
Already up-to-date.



Generated by build [OMERO-DEV-latest-docs-autogen#1058](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1058/).

----
--no-rebase

jburel commented:
Chicken and egg problem
We will have to keep the manual copy of the files added https://github.com/openmicroscopy/ome-documentation/pull/1761
or roll back the change in webinstall
and re-open a PR then 1761 is merged.

I will vote for the first option since it is already brutal enough to synch the various repo for the doc.
cc @sbesson

pwalczysko commented:
Finished testing of https://github.com/openmicroscopy/ome-documentation/pull/1761

snoopycrimecop committed:
Update auto-generated documentation

Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by OMERO-DEV-latest-docs-autogen#1058 (https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1058/)
